src/renderer/gdi/lib/sources

```diff
@@ -6,3 +6,4 @@
 
 TARGETNAME              = ConRenderGdi
 TARGETTYPE              = LIBRARY
+TARGETLIBS              = $(SDK_LIB_PATH)\usp10.lib

```

Comment:
Unfortunately, this won't work because ConRenderGdi is a static lib itself. Windows' build system doesn't roll up `TARGETLIBS` from dependency projects.

Your best bet will be to find _any `sources` file_ that mentions `ConRenderGdi.lib` by name and add this to its existing `TARGETLIBS`.